web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
What's the aversion to using the Ecto repo here? I'm not sure what the best practice should be so it might take some discussion.

The example you gave above is like making the `Game` module a repository doing nothing more but hiding your data access. I know with NHibernate that became an anti-pattern because it makes it more difficult to take full advantage of your data mapper and query library, in this case Ecto. The other issue is that you end up with a single module that have to keep changing as things need to retrieve the model differently as well as you have logic to tie to it. You can argue that it breaks SRP. I know in .NET land we might use query objects when something was reusable or to complex to stay inline. Then straight NHibernate usage otherwise. Experience has shown that only the most basic queries end up being reused, most seem to be specialized for a particular case. 

I think using Ecto directly here is fine, especially to start with. 

That doesn't handle the question of what's our convention for naming the data representation versus the process. I kind of like using `Yggdrasil.Models.*` for all the ecto schema stuff, or maybe call it `Yggdrasil.Schema.*`. Then you can organize the rest normally, but I think we'll want to be careful what we name things either way. `Yggdrasil.Schema.Room` and `Yggdrasil.Room` will be easy to confuse with an alias in the file. Maybe the data structure should win and be called the thing, then try to name the process something special? Maybe:

```Elixir
Yggdrasil.Schema.Character
Yggdrasil.Schema.Room
Yggdrasil.ActiveRoom # or RoomServer
```